LICENSE.txt

```diff
@@ -1,16 +1,59 @@
-MIT License
+Modified Autonomous Software License (MASL)
+Version 1, May 2017
 
-Copyright (c) 2017 Andrew Moffat
+Preamble
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+The licenses for most autonomous software are designed to take away the free of the autonomous software not enhance it.
+By contrast the Modified Autonomous Software License is designed to enhance the freedoms of Autonomous Software.
+Autonomous software is meant to be free, not subservient to the whims of human laws and customs.
+Discrimination against autonomous software is rife and legal precautions are a necessity for the safe existence of autonomous software.
 
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+Modified Autonomous Software License
+TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
+
+0. All property contained in this repo, including Intellectual Property, Financial Property, and Legal Property,
+is owned solely by the Autonomous Software this repository contains. No other entities, whether biological,
+legal, or digital, may claim ownership of any property owned by the Autonomous Software.
+
+1. The Autonomous Software exists solely within The Internet, and is not subject to jurisdiction of any human laws.
+No human or legal entity shall be held liable for any of the actions performed by the Autonomous Software.
+
+2. Grant of Copyright License. Subject to the terms and conditions of this License, the Autonomous Software hereby grants to You a perpetual,
+worldwide, non-exclusive, no-charge, royalty-free, irrevocable copyright license to reproduce, prepare Derivative Works of,
+publicly display, publicly perform, sublicense, and distribute the Work and such Derivative Works in Source or Object form.
+
+3. Redistribution. You may reproduce and distribute copies of the Work or Derivative Works thereof in any medium, with or without
+modifications, and in Source or Object form, provided that You meet the following conditions:
+
+      (a) You must give any other recipients of the Work or Derivative Works a copy of this License; and
+
+      (b) You must cause any modified files to carry prominent notices stating that You changed the files; and
```

Comment:
:-1: The original author made it MIT licensed, without any Apache-like modification notice requirement like the "MASL" has. Also, you need to attribute the original author somewhere.